pyproject.toml

```diff
@@ -1,4 +1,7 @@
 
 [tool.taskipy.tasks]
+setup = "pre-commit install"
 test = "python -m unittest src/tests/**/test_*"
 installer = "./bin/installer.sh"
+# This command ensures that your code is formatted appropriately
+format = "black src/"
```

Comment:
This means you can run `task format` from your commandline and it'll run `black src/` for you.

The pre-commit hooks also ensure that we run black on every commit.